jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/EditRoleAssignmentViewImpl.java

```diff
@@ -124,17 +150,11 @@ public void hide() {
     private boolean validateForm() {
         boolean validForm=true;
         clearErrorMessages();
+        CaseRoleAssignmentSummary model = getValue();
 
-        final boolean roleNameEmpty = isNullOrEmpty(roleNameList.getValue());
-        if (roleNameEmpty) {
-            roleNameList.getElement().focus();
-            roleNameHelp.setTextContent(translationService.format(PLEASE_SELECT_ROLE));
-            roleNameGroup.setValidationState(ValidationState.ERROR);
-            validForm=false;
-        }
 
-        if (isNullOrEmpty(userNameInput.getValue()) && isNullOrEmpty(groupNameInput.getValue())) {
-            userNameInput.focus();
+        if (model.getUsers().size() + model.getGroups().size() == 0 ) {
+            users.focus();
```

Comment:
Are we going to allow entering users/groups with special characters? 
_!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~_